hikari/impl/bot.py

```diff
@@ -194,30 +195,11 @@ class GatewayBot(traits.GatewayBotAware):
            we should also chunk small guilds if the presences are not declared).
         4. The members cache is enabled or there are listeners for the
            `MemberChunkEvent`.
-    logs : typing.Union[None, LoggerLevel, typing.Dict[str, typing.Any]]
+    logs : typing.Union[None, str, int, typing.Dict[str, typing.Any], os.PathLike]
         Defaults to `"INFO"`.
 
-        If `None`, then the Python logging system is left uninitialized
-        on startup, and you will need to configure it manually to view most
-        logs that are output by components of this library.
-
-        If one of the valid values in a `LoggerLevel`, then this will match a
-        call to `colorlog.basicConfig` (a facade for `logging.basicConfig` with
-        additional conduit for enabling coloured logging levels) with the
-        `level` kwarg matching this value.
-
-        If a `typing.Dict[str, typing.Any]` equivalent, then this value is
-        passed to `logging.config.dictConfig` to allow the user to provide a
-        specialized logging configuration of their choice. If any handlers are
-        defined in the dict, default handlers will not be setup.
-
-        As a side note, you can always opt to leave this on the default value
-        and then use an incremental `logging.config.dictConfig` that applies
-        any additional changes on top of the base configuration, if you prefer.
-        An example of can be found in the `Example` section.
-
-        Note that `"TRACE_HIKARI"` is a library-specific logging level
-        which is expected to be more verbose than `"DEBUG"`.
+        Have a look at `hikari.internal.ux.init_logging`'s `flavour` parameter for more info
```

Comment:
linking to internal functions which enders users shouldn't be touching sounds like a questionable move

Reply:
Changed